components/datepicker/stories/datepicker.stories.js

```diff
@@ -34,15 +33,15 @@ export default {
 			},
 			control: "boolean",
 		},
-		isValid: {
-			name: "Valid",
+		isDateTimeRange: {
+			name: "Date Time Range styling",
```

Comment:
```suggestion
			name: "Range",
```

Reply:
There is separate styling for displaying the date and time but that's not really clear with how I've named this, updating it.